app/Main.hs

```diff
@@ -51,11 +54,16 @@ randomNums x = do
 getAnswer :: String -> Maybe String
 getAnswer x = do
   y <- readMaybe x
-  Prelude.lookup y baseNumbers
+  countInJapanese y
 
+-- `numberWang` generates a digit number (1 to 5) and
+-- using that a random quiz number.
+-- If the guess is equal to the actual translation THAT'S NUMBERWANG!
 numberWang :: IO ()
 numberWang = do
-  num <- randomNums 1
+  g <- newStdGen
+  let h = take 1 (randomRs ('1', '5') g)
```

Comment:
I think this "1" should be an argument of numberWang. In it's current state it's not possible to work with numbers in range, say 1-1000.